source/php/Index.php

```diff
@@ -330,6 +330,8 @@ public function getTheExcerpt($post, int $numberOfWords = 55) {
                 $excerpt .= render_block($block) . " " . PHP_EOL; 
             }
         }
+        
+        $excerpt = preg_replace('/\[(.*?)\]/', '', $excerpt);
```

Comment:
Will this also match shortcodes with arguments? `[foobar foo=bar]`

Reply:
Yes, according to chatgpt ;)